README.md

```diff
@@ -21,6 +21,11 @@ All required dependencies are listed in this [conda environment file](environmen
 
 1. (Suggested) Create a new conda environment based on the provided [YAML file](environment.yaml).
   - Run `conda env create --name avalon --file=environment.yaml`
+  - Install the ipykernel:
+```
+conda activate avalon
+python -m ipykernel install --user --name avalon --display-name "Python (avalon)"
+```
   - Then switch to `avalon` env by clicking the drop-down, and select `avalon`
```

Comment:
What's this part for , may I ask?

Reply:
We have to run this in order for Jupyter Lab to recognize the kernel of the new conda environment

Reply:
Here is the documentation on this:
https://ipython.readthedocs.io/en/stable/install/kernel_install.html

Reply:
Ok, nice. I usually just refresh the web page. thank you for explanation.

Reply:
OK thanks, I don't know why but my jupyter lab recognize the kernel in this way only, the reloading does not work for me.